services/scan/src/precinct_scanner_app.ts

```diff
@@ -440,13 +453,26 @@ export async function buildPrecinctScannerApp(
 
   app.post<NoParams, Scan.ExportResponse, Scan.ExportRequest>(
     '/precinct-scanner/export',
-    (_request, response) => {
-      const outputStream = new streams.WritableStream();
-      store.exportCvrs(outputStream);
-      const cvrs = outputStream.toString();
+    (request, response) => {
+      const skipImages = request.body?.skipImages;
+      debug(`exporting CVRs ${skipImages ? 'without' : 'with'} inline images`);
+
+      const cvrFilename = generateFilenameForScanningResults(
+        // TODO: Move machine config provider to shared utilities and access
+        // actual machine config, with dev overrides, here instead
+        '0000',
+        getBallotsCounted(store),
+        store.getTestMode(),
+        new Date()
+      );
```

Comment:
this is really a placeholder, and could be removed, because the frontend is overriding the filename in the content-disposition

Reply:
maybe it's bad to have it in for now because if the `TODO` isn't done and and the endpoint is relied on later it could give CVR filenames without correct machine ID